### PR TITLE
[INTRODUCTION] Adding additional chatbots

### PR DESCRIPTION
**Pull request by ehumph**
Added CodeWhisperer and CoPilot to the intro section on chatbots and bolded the fact that we do not endorse any particular bot. Also added a dino cartoon and simplified the introduction

### CODE REVIEW

**Comment by ehumph** [comment]
I got cocky and thought I was starting to master github, and github actions quickly smacked me down.

**Comment by avahoffman** [comment]
> I got cocky and thought I was starting to master github, and github actions quickly smacked me down. Like Icarus, I flew too close to the sun.

It likes to do that :P

**Review by avahoffman — APPROVED** [review]
Looks good! Minor note about the order of the callout boxes.

File: 01-intro.Rmd [hunk 1]
```diff
@@ -50,20 +57,46 @@ We must always be aware of the potential for harm and deliberately take steps to
...
+:::
+
+
+:::{.ethics}
+In this box, you'll find reminders of the ethical considerations you should be particularly aware of while using AI to do certain tasks. 
+:::
+
+Wherever possible, we use screenshots of the actual answers we got from the AI chatbots in response to our questions. Occasionally when we have asked for the AI to write or fix code, we have formatted the answer in such a way that makes it easier to use. In these instances, the AI answer will look like this:
+
+:::{.ai_response}
```
[the remark this answers is not in the document]
Comment (ehumph) [reply 01-intro.Rmd:89]: That makes a lot of sense.

**Pull request merged**